DFC.ServiceTaxonomy.CustomEditor/Views/ContentEdit_AdditionalActions.cshtml

```diff
@@ -13,37 +13,46 @@
     var hasDraft = contentItem.HasDraft();
     var hasEditPermission = await AuthorizationService.AuthorizeAsync(User, CommonPermissions.EditContent, contentItem);
```

Comment:
we dont need to call the authorisation service if the content type is footer or header, can this be done in a if else statment?

Reply:
Updated to use a coalescing operator instead.